dist/v2.1.0/account-info-nz-swagger.yaml

```diff
@@ -12,8 +12,8 @@ info:
   license:
     name: Licence
     url: "https://www.apicentre.paymentsnz.co.nz/join/"
-  version: v2.1.0
-basePath: /open-banking-nz/v2.1
+  version: v2.2.0
```

Comment:
Is the version bump from v2.1.0 to v2.2.0 a typo for this v2.1.0 swagger definition?